ivy.el

```diff
@@ -3227,12 +3227,20 @@ Otherwise, ~/ will move home."
 
 (defvar ivy--exhibit-timer nil)
 
+(defalias 'ivy-input-changed-p
```

Comment:
Please add docstrings for new definitions following the conventions described under [`(info "(elisp) Documentation Tips")`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Documentation-Tips.html).

Is this intended as an internal function or part of a public API?  In the former case its name should have two hyphens, i.e. `ivy--input-changed-p`.

Reply:
If you ask me, I'd say this is somewhat of a bare-bone mesh up, it should be kept as an internal 

I'd like ask you kindly to make the decision instead since clearly you are the more experienced one.

Reply:
As I was thinking about this question, I digged a bit more into the ivy api and the tested the behaviour a little more.

It occurred to me the predicate could be further improved to use the settings in `ivy-more-chars-alist`, where `ivy-input-changed-p` would return `nil` to force immediate updates apropreiately depending on current user input text length and `ivy-more-chars-alist`.

Prior to this, this decision is "muffled" to me, as I thought this could potentially be useful, but now it has been more tailored to `ivy--queue-exhibit` I am more inclined to keep it as an internal function.

Reply:
> I'd like ask you kindly to make the decision instead since clearly you are the more experienced one.

The function looks like a helper function that is not intended to be used outside of Ivy, which makes it internal. If, for whatever reason, it becomes useful outside of Ivy, it's trivial to rename an internal function to a public one, whereas the opposite is not.

Tl;dr go with double hyphen `--`.

Reply:
alright